src/AvaloniaEdit.TextMate/AvaloniaEdit.TextMate.csproj

```diff
@@ -12,8 +12,9 @@
 
   <ItemGroup>
     <PackageReference Include="Avalonia" Version="$(AvaloniaVersion)" />
-    <PackageReference Include="TextMateSharp" Version="$(TextMateSharpVersion)" />
-    <PackageReference Include="TextMateSharp.Grammars" Version="$(TextMateSharpVersion)" />
+    <PackageReference Include="Avalonia.Themes.Fluent" Version="11.0.0" />
```

Comment:
Can we revert these changes from non-demo project?
Controls shouldn't depend on a single specific theme.